src/mount.ts

```diff
@@ -133,6 +137,46 @@ export function mount(originalComponent: any, options?: MountingOptions) {
   const { emitMixin, events } = createEmitMixin()
   vm.mixin(emitMixin)
 
+  // stubs
+  if (options?.global?.stubs) {
+    transformVNodeArgs((args) => {
+      // regular HTML Element. Do not stubs these
+      if (Array.isArray(args) && typeof args[0] === 'string') {
+        return args
+      }
+
+      // don't care about comments/fragments
+      if (typeof args[0] === 'symbol') {
+        return args
+      }
+
+      // do not stub the VTU Parent component
+      if (typeof args[0] === 'object' && args[0]['name'] === 'VTU_COMPONENT') {
+        return args
+      }
+
+      if (
+        typeof args[0] === 'object' &&
+        args[0]['name'] in options?.global?.stubs
+      ) {
+        const name = args[0]['name']
+        // default stub
+        if (options?.global?.stubs[name] === true) {
+          return [createStub({ name: args[0]['name'] })]
+        }
```

Comment:
@dobromir-hristov we one of us gets to working on `findComponent` we will probably need to pass `props` as the second arg here, so we can do `findComponent(Foo).props)`. So it will be like 

`return [createStub({ name: args[0]['name'] }), args[1]]` or something. I didn't do it here because I am not exactly sure how this will work yet, but I don't expect it to be a blocker.

Reply:
Yep, we will check it out.